torch/csrc/jit/fuser/cuda/resource_strings.h

```diff
@@ -13,6 +13,25 @@ tensor as input. Correct code for this case is generated, however, nvrtc does
 not know how to handle int*_t integer types, so typedefs help it handle those
 cases*/
 
+#ifdef __HIP_PLATFORM_HCC__
+static auto type_declarations_template = CodeTemplate(R"(
+#include <hip/hip_runtime.h>
+${HalfHeader}
+${RandHeader}
+
```

Comment:
I think you want the `POS_INFINITY` and `NEG_INFINITY` constant definitions (maybe just to `INFINITY` and `-INFINITY`) to pass the `TestFuser.test_clamp` test. (Of course, one might see if we strictly need to emit `(a < INFINITY ? a : INFINITY)`, but that's a different change.)

Reply:
Spot on, thanks!